Assets/Tests/Performance/Runtime/HeadlessBenchmark/Scripts/HeadlessBenchmark.cs

```diff
@@ -1,13 +1,13 @@
 using System;
 using System.Collections;
-using Mirror.KCP;
+using Mirage.KCP;
 using UnityEngine;
 using Cysharp.Threading.Tasks;
 #if IGNORANCE
-using Mirror.ENet;
+using Mirage.ENet;
```

Comment:
```suggestion
using Mirror.ENet;
```
this might need to stay as mirror till ignorance for NG has been changed to Mirage